doc/api/process.md

```diff
@@ -2270,6 +2307,7 @@ cases:
 [Readable]: stream.html#stream_readable_streams
 [Signal Events]: #process_signal_events
 [Stream compatibility]: stream.html#stream_compatibility_with_older_node_js_versions
+[`throw`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw
```

Comment:
Nit: it seems this needs to be placed after ``[`subprocess.kill()`]: ... ``, alphabetically.